app/celery/tasks.py

```diff
@@ -176,7 +175,7 @@ def send_sms(self, service_id, notification_id, encrypted_notification, created_
 
 
 @notify_celery.task(name="send-email")
-def send_email(service_id, notification_id, encrypted_notification, created_at, reply_to_addresses=None):
```

Comment:
Removing the reply_to_addresses may be not compatible with the previous versions of tasks created. What happens if an old task is in the queue with reply_to_addresses and is picked up by a new task. Will that work?